intro-HPC/ch_introduction.tex

```diff
@@ -109,35 +113,64 @@ \section{What is the \hpc?}
-    \item  48 compute nodes with 2 4-core AMD 2378 (Shanghai) processors, 32 GB memory,
-           72 GB local disk, SDR-IB network
-    \item  64 compute nodes with 2 8-core AMD 6134 (Magnycours) processors, 64 GB memory,
-           1 TB local disk, QDR-IB network
-    \item  9 compute nodes with 2 16-core AMD 6274 (Interlagos) processors, 256 GB memory,
-           900 GB local disk, QDR-IB network
-    \item  29 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64 GB memory,
-           900 GB local disk, QDR-IB network
-    \item  6 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64 GB memory,
-           900 GB local disk, QDR-IB network, 2 Tesla K20x NVIDIA GPGPUs with 6Gb memory 
-    \item  2 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64 GB memory,
-           900 GB local disk, QDR-IB network, 2 Intel Xeon Phi 5110P coprocessors with Gb memory
+    \item  48 compute nodes with 2 4-core AMD 2378 (Shanghai) processors, 32 GB
+    memory, 72 GB local disk, SDR-IB network
+    \item  64 compute nodes with 2 8-core AMD 6134 (Magnycours) processors, 64
+    GB memory, 1 TB local disk, QDR-IB network
+    \item  9 compute nodes with 2 16-core AMD 6274 (Interlagos) processors, 256
+    GB memory, 900 GB local disk, QDR-IB network
+    \item  29 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64
+    GB memory, 900 GB local disk, QDR-IB network
+    \item  6 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64 GB
+    memory, 900 GB local disk, QDR-IB network, 2 Tesla K20x NVIDIA GPGPUs with
+    6Gb memory
+    \item  2 nodes with 2 10-core INTEL E5-2680v2 (IvyBridge) processors, 64 GB
+    memory, 900 GB local disk, QDR-IB network, 2 Intel Xeon Phi 5110P
+    coprocessors with Gb memory
   \end{enumerate}
 \fi
+\ifgent
+  \begin{enumerate}
+    \item[gastly] 46 compute nodes with 2 4-core Intel L5520 (Nehalem)
+    processors, 12 GB memory, 90 GB local disk, GBe network
+    \item  haunter: 151 compute nodes with 2 4-core Intel L5520 (Nehalem)
+    processors, 12 GB memory, 90 GB local disk, GBe network
+    \item  gulpin: 32 compute nodes with 2 8-core AMD Opteron 6136
+    (Magny-Cours) processors, 64 GB memory, 483 GB local disk, Double QDR-IB
+    network
+    \item  dugtrio: 16 compute nodes with 2 6-core Intel X5675 (Westmere-EP)
+    processors, 96 GB memory, 483 GB local disk, Double QDR-IB network. This is
+    a Symmetric Multi-Processing (SMP) machine based on ScaleMP\'s vSMP
+    Foundation (Reverse virtualization)
+    \item  raichu: 64 compute nodes with 2 8-core Intel E5-2670 (Sandy Bridge)
+    processors, 32 GB memory, 400 GB local disk, GBe network \item  delcatty:
+    160 compute nodes with 2 8-core Intel E5-2670 (Sandy Bridge) processors, 32
+    GB memory, 400 GB local disk, FDR-IB network
+    \item  phanpy: 16 compute nodes with 2 12-core Intel E5-2680v3 (Haswell-EP)
+    processors, 512 GB memory, 3X 400 GB local disk (SSD), Dual FDR-IB network
```

Comment:
we planm to make the SSDs avaialble as one big blockdevice, so make it a 1.2TB SSD